src/tools/bin/plot_chargeConservation.py

```diff
@@ -89,7 +89,7 @@ def plotError(h5file, slice_pos=[0.5, 0.5, 0.5]):
     charge = np.zeros_like(Ex)
     norm = 0.0
     for field_name in f["/data/{}/fields/".format(timestep)].keys():
-        if field_name[0:8] == "Density_":
+        if field_name[1:15] == "_chargeDensity":
```

Comment:
This will not work for species names longer than one character. Let me fix this tomorrow.